packages/trader/src/Modules/Page404/Components/__tests__/Page404.spec.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Router } from 'react-router';
+import { createBrowserHistory } from 'history';
+import { render } from '@testing-library/react';
+import Page404 from '../Page404';
+
+describe('Page404', () => {
+    const browser_history = createBrowserHistory();
+
+    it('should render Page404', () => {
+        const { container } = render(
+            <Router history={browser_history}>
+                <Page404 />
+            </Router>
+        );
+
+        expect(container).toBeInTheDocument();
```

Comment:
@kate-deriv unfortunately, this test will always pass because container is a div inside which <Router is rendered, so in this test case we are not testing the rendering of <Page404. To test the rendering of <Page404, please try the following:
```suggestion
        expect(screen.getByText('We couldn’t find that page')).toBeInTheDocument();
```

Reply:
Noted, thank you